codespell_lib/data/dictionary.txt

```diff
@@ -6984,7 +7314,9 @@ cofrimations->confirmations
 cofrimed->confirmed
 cofriming->confirming
 cofrims->confirms
+cogniscent->cognizant
```

Comment:
In the UK usually `cognisant`.
Add `cognisant->cognizant` to GB-US dictionary by the way...

Reply:
> In the UK usually cognisant.

Hmm, maybe it's worth adding as an alternative, but according to [Grammarhow.com](https://grammarhow.com/cognisant-or-cognizant/):

> The [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/cognizant) states that “cognisant” is a UK variant of “cognizant.”
>
> The [Google Ngram](https://books.google.com/ngrams/graph?content=cognisant%2Ccognizant&year_start=1900&year_end=2019&corpus=29&smoothing=3&case_insensitive=true) shows that from the start of the 20th century to the 1960s, “cognisant” was more common in the UK than “cognizant.”
>
> However, since the 1960s, as with many other words, “cognizant” has become the preferred version and the use of “cognisant” has been in almost constant decline since the 1920s.

[Oxford Learner's Dictionary](https://www.oxfordlearnersdictionaries.com/definition/english/cognizant) lists "cognisant" as a variant in British English.

Google hits:
[cognizant](https://www.google.com/search?q=cognizant) approx. 76 million
[cognisant](https://www.google.com/search?q=cognisant) approx. 3 million

What do you think?

>  Add cognisant->cognizant to GB-US dictionary by the way...

Done.

Reply:
`cognisant` might still be preferred by some British users. That's a rare word, those who use it are probably cognisant enough to sort this out :smile: Having both seems OK to me.

Reply:
> That's a rare word, those who use it are probably cognisant enough to sort this out 😄 Having both seems OK to me.

That's true! 😄